server/chat-plugins/room-events.ts

```diff
@@ -18,11 +18,28 @@ function formatEvent(event: {eventName: string, date: string, desc: string, star
 	}
 	let ret = `<tr title="${explanation}">`;
 	ret += Chat.html`<td>${event.eventName}</td>`;
+	ret += Chat.html`<td>${(event.aliases ? event.aliases.join(", ") : "")}</td>`;
 	ret += `<td>${Chat.formatText(event.desc, true)}</td>`;
 	ret += Chat.html`<td><time>${event.date}</time></td></tr>`;
 	return ret;
 }
 
+function getAllAliases(room: Room) {
+	return Object.keys(room.events).map(event => room.events[event].aliases).join();
+}
+
+function getEventIDByNameOrAlias(nameOrAlias: string, room: Room): ID {
```

Comment:
I think this can just be called `getEventID`, its obviously grabbing the ID via alias or name with the name of the first argument and how it's being used throughout the code.